### ISSUE TITLE
[Question] How to set the icon for a targets which displays next to the target in Xcode

### ISSUE THREAD

**Issue by jakemumu** [CocoaPods/Xcodeproj#875]
Hey! I was trying this for a few hours last night and couldn't get it working -- assuming I have a valid Icon.icns -- is there a way I could set that icon for the target to display next to the target in Xcode? 

I'm working on an Xcodeproj which has a ton of targets and this would feel really nice for my dev team when working if I can get it working.

**Comment by jakemumu** [comment]
Hmm -- yeah it's a CMake generated Xcode project I'm cleaning with XcodeProj cause all the junk they made -- it looks like this was set on the target but perhaps I'll try removing and reading here -- will report back if successful, thanks for the quick reply

**Comment by jakemumu** [comment]
Could I ask another question by chance @dnkoutso  -- is there a way to Alpha sort the scheme/target list as displayed in the dropdown when selecting targets to build? 

Sorting the targets is really nice but it doesn't appear to update the dropdown

**Comment by dnkoutso** [comment]
I dont think so thats probably Xcode functionality, it might depend on scheme name being alphabetic or the order the schemes are referenced in the .xcodeproj.
